Automattic/wp-calypso
# Add Calypso localhost development environment

### Description

Adds the ability for the WP-Desktop app to interact with Caypso at `localhost:3000`. Much thanks to @belcherj for figuring out how to get this working! 

### To Use

(1) Run Calypso with `yarn start`,
(2) run desktop (from desktop folder) with `yarn run dev:localhost`


### To Test

Verify that IPC is working by selecting `IPC: Ping` from the `View` menu (Calypso should respond with `Pong`).

<p align="center">
<img width="233" alt="view-menu-ping" src="https://user-images.githubusercontent.com/8979548/116316077-aa47b780-a76e-11eb-9b6f-09effbef10e3.png">
<img width="365" alt="calypso-says-pong" src="https://user-images.githubusercontent.com/8979548/116316082-ab78e480-a76e-11eb-8917-ef6a48c9bb23.png">
</p>
